CMakeLists.txt

```diff
@@ -128,6 +128,10 @@ else()
   endif()
   list(APPEND COMPUTECPP_USER_FLAGS "-Xclang -cl-mad-enable")
   list(APPEND COMPUTECPP_USER_FLAGS "-Xclang -cl-no-signed-zeros")
+  if (MSVC)    
+    # Enable Exception Handling Semantics in all configurations
+    add_compile_options(/EHSc)
```

Comment:
Is this a ComputeCpp specific thing? Should it be moved outside of this `if` block? Or should it be added to each target individually in [`snn_target`](https://github.com/codeplaysoftware/SYCL-DNN/blob/50c3a38e79646b1dae25337d1642e42c826c0d15/cmake/SNNHelpers.cmake#L174)?